src/modules/custom/box.rs

```diff
@@ -20,9 +20,7 @@ impl CustomWidget for BoxWidget {
         let container = build!(self, Self::Widget);
 
         if let Some(orientation) = self.orientation {
-            container.set_orientation(
-                try_get_orientation(&orientation).unwrap_or(Orientation::Horizontal),
-            );
+            container.set_orientation(Orientation::from(orientation));
```

Comment:
Implementing `From` gives you a free `Into` impl, and Rust is clever enough to infer the types. Sometimes using From is better because it's more explicit, but I think here it's obvious enough the concise syntax is preferable: 

```suggestion
            container.set_orientation(orientation.into());
```

(Same story in `slider` and `sysinfo`)

Reply:
Thanks.